petastorm/pytorch.py

```diff
@@ -133,7 +135,8 @@ def __init__(self, reader, batch_size=1, collate_fn=decimal_friendly_collate,
         if shuffling_queue_capacity > 0:
             # We can not know what is the reasonable number to use for the extra capacity, so we set a huge number
             # and give up on the unbound growth protection mechanism.
-            min_after_dequeue = shuffling_queue_capacity - 1
+            if not min_after_dequeue:
```

Comment:
I think, if `min_after_dequeue` = 0, we would override the value, which is not the intent.